teensy3/src/lib.rs

```diff
@@ -18,11 +18,11 @@ pub mod util;
 
 /// Processor panic
 #[lang = "panic_fmt"]
-pub extern fn rust_begin_panic(msg: core::fmt::Arguments, file: &'static str, line: u32) -> ! {
+pub extern "C" fn rust_begin_panic(msg: core::fmt::Arguments, file: &'static str, line: u32) -> ! {
```

Comment:
Why are these marked as C now?
Curious because I've not had a problem with this before.